### DISCUSSION TITLE
Svelte support

### DISCUSSION THREAD

**Discussion by AsuraKev** [algolia/instantsearch#5869] [General] [answered]
Hi there,

With the rise of Svelte and Sveltekit, will there be a version of instant search of Svelte? :)

**Comment by Haroenv** [comment, marked as answer]
For now this isn't yet planned, but our engineer @aymeric-giraudet has created a community version: https://github.com/aymeric-giraudet/svelte-algolia-instantsearch

**Comment by Gustutu** [comment]
also interested in a svelte version :)

**Reply by Haroenv** [reply]
Have you checked https://github.com/aymeric-giraudet/svelte-algolia-instantsearch ?

**Comment by mathg** [comment]
Hello, do you have any update on this discussion? We have used the Aymeric's version for couple projects but no updates were made in the last 8 months. Do you plan to officially support Svelte? Thanks!

**Reply by Haroenv** [reply]
Were any updates needed? I know the recommend components weren't added yet as nobody had asked for them; is there anything else missing?

**Reply by mathg** [reply]
There were couple issues that needed to be resolved and pull requests that could be merged to improve some small features. Most of the things are there like you said, just some minor improvements could be done.

**Reply by mathg** [reply]
@Haroenv another issue that we got since algoliasearch is not on version 5

└─┬ svelte-algolia-instantsearch 0.12.1
  └── ✕ unmet peer algoliasearch@">= 3.1 < 5": found 5.5.2

**Reply by Haroenv** [reply]
@mathg you can make a pull request to change the peer dependency then
